config/systemd-monitor.json

```diff
@@ -0,0 +1,28 @@
+{
+	"plugin": "journald",
+	"pluginConfig": {
+		"source": "systemd"
+	},
+	"logPath": "/var/log/journal",
+	"lookback": "",
+	"bufferSize": 10,
+	"source": "systemd-monitor",
+	"conditions": [],
+	"rules": [
+		{
+			"type": "temporary",
+			"reason": "KubeletRestart",
+			"pattern": "Started Kubernetes kubelet."
+		},
+		{
+			"type": "temporary",
+			"reason": "DockerRestart",
+			"pattern": "Starting Docker Application Container Engine..."
```

Comment:
BTW, Seems that this log content is only adapted to GKE, right? If we want to detecte a `DockerRestart` event, we need to change the `pattern`. Since we can not add comment to a json file, Maybe `GKE` prefixed event name is more clear. How about `GKEKubeletStart`/`GKEDockerRestart`.